scripts/linux/install-kubernetes.sh

```diff
@@ -15,19 +15,17 @@ done
 
 ansible_debug="$1"
 
-echo "Ansible debug: $ansible_debug"
+echo "Ansible debug flag: $ansible_debug"
```

Comment:
echo "ansible_debug: $ansible_debug"

Reply:
agreed and changed ! (also in Vagrantfile)

Reply:
done